drivers/dma/dma_stm32.c

```diff
@@ -304,7 +313,6 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 		periph_addr_adj = config->head_block->dest_addr_adj;
 		break;
 	/* Direction has been asserted in dma_stm32_get_direction. */
-	}
```

Comment:
Is this a typo?

Reply:
yes, fixed elsewhere, now fixed in this commit